app/src/main/java/com/iic/shopingo/ui/trip_flow/views/ShoppingListItem.java

```diff
@@ -0,0 +1,113 @@
+package com.iic.shopingo.ui.trip_flow.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.CheckBox;
+import android.widget.FrameLayout;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import com.iic.shopingo.R;
+import com.iic.shopingo.ui.trip_flow.activities.ShoppingListActivity;
+import com.iic.shopingo.ui.trip_flow.data.ShoppingList;
+
+/**
+ * Created by asafg on 04/03/15.
+ */
+public class ShoppingListItem extends FrameLayout {
```

Comment:
It messed up the layout preview.
I'd rather have a working preview than a slightly more optimal view tree.